File: crud-python/Ex03_Insert_Journey.py
#!/usr/bin/env python3
from db_connection import Connection
import uuid

print('========================================')
print('Start exercise')

# journey_id = uuid.uuid1()
journey_id = uuid.UUID('230995ee-c697-11ea-b7a1-8c85907c08dd')
spacecraft_name = 'Crew Dragon Endeavour,SpaceX'

try:
    connection = Connection()
    output = connection.session.execute(
        "INSERT INTO spacecraft_journey_catalog (spacecraft_name, journey_id, active, summary) VALUES (%s,%s,%s,%s)",
        [spacecraft_name, journey_id , False,'Bring Astronauts to ISS']
    )
except Exception as e: 
    print(e)
    print('Failure')
else:
    print('Journey created ', journey_id)
    print('Success')
    print('Closing connection (up to 10s)')
finally:
    connection.close()
print('========================================')

File: crud-nodejs/Ex03_Insert_Journey.js
const connection = require('./db_connection')
const Uuid       = require('cassandra-driver').types.Uuid;
const TimeUuid   = require('cassandra-driver').types.TimeUuid;

// this is an insert statement in nodejs
//const journey_id = TimeUuid.now();
const journey_id = TimeUuid.fromString('84121060-c66e-11ea-a82e-f931183227ac')
const spacecraft_name = 'Crew Dragon Endeavour,SpaceX';

const insert     = 'INSERT INTO spacecraft_journey_catalog (spacecraft_name, journey_id, active, summary) VALUES (?,?,?,?);';
const params     = [spacecraft_name, journey_id ,false,'Bring Astronauts to ISS'];

console.log("========================================")
console.log("Start exercise")

connection.client.execute(insert, params)
.then(function (result){
	console.log("Journey created : %s", journey_id.toString())
    console.log("SUCCESS")
    connection.client.shutdown()
})
.catch(function (error){
     console.log(error.message)
     connection.client.shutdown()
});

console.log("========================================")